### PR TITLE
remove debugger only code path from bluebird

### PR DESCRIPTION
**Pull request by makarandp0**
Note: This PR is against a staging branch...bluebird3, and not master. The staging branch will merge into master later when all the changes in Bluebird as well as Hadron repo are fully baked. 
 
Bluebird 3 behaves differently when running under chrome debugger. This makes it really difficult to debug issues that only repros w/o debugger! to keep things predictable I am turning off this special code path. 

Basically with debugger available BlueBird decided to execute the code w/o adding it to the taskpool, where as when not running in debugger it goes into task pool for processing later. With this change we won't see nice callstack when running under debugger, but I think its better this way, since that nice callstack was artificial  anyways.

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -363,8 +363,9 @@ var ret = {
     markAsOriginatingFromRejection: markAsOriginatingFromRejection,
     classString: classString,
     copyDescriptors: copyDescriptors,
-    hasDevTools: typeof chrome !== "undefined" && chrome &&
-                 typeof chrome.loadTimes === "function",
+    hasDevTools: false,
+    // hasDevTools: typeof chrome !== "undefined" && chrome &&
+    //              typeof chrome.loadTimes === "function",
```
Comment (finnigantime) [inline src/util.js]: Seems fine for now. So it sounds like this is a code path for resolving promises sync rather then resolving them async using the bluebird scheduler. Seems like that may be valuable for us for debugging errors in promise callstacks - it will allow us to get bluebird off the stack and perhaps keep more context from the previous Hadron stack. Also, seems like a valuable mechanism (along with batchSize) for helping flush out race conditions in tests.

Can you file a separate task (and tag here) to add a configuration param for enabling that code path from Hadron?

Comment (finnigantime) [inline src/util.js]: You can link to `HAD-1673 Add artificial delay to promises to weed out timing issues causing test instability`

**Comment by finnigantime** [comment]
1 request for task, otherwise LGTM

**Comment by makarandp0** [comment]
thats good Idea @finnigantime, I will add a TODO referencing this task.
